docs/spec/spec.md

```diff
@@ -0,0 +1,556 @@
+# Specification: Ballerina FTP Library
+
+_Owners_: @shafreenAnfar @dilanSachi @Bhashinee    
+_Reviewers_: @shafreenAnfar @Bhashinee  
+_Created_: 2020/10/28   
+_Updated_: 2022/02/07  
+_Edition_: Swan Lake    
+_Issue_: [#2202](https://github.com/ballerina-platform/ballerina-standard-library/issues/2202)
+
+# Introduction
+This is the specification for the FTP standard library of [Ballerina language](https://ballerina.io/), which provides FTP client/listener functionalities to send and receive files by connecting to a FTP/SFTP server.
+
+The FTP library specification has evolved and may continue to evolve in the future. Released versions of the specification can be found under the relevant Github tag.
+
+If you have any feedback or suggestions about the library, start a discussion via a Github issue or in the [Slack channel](https://ballerina.io/community/). Based on the outcome specification and implementation can be updated. Community feedback is always welcome. Any accepted proposal which affects the specification is stored under `/docs/proposals`. Proposals under discussion can be found with the label `type/proposal` in Github.
```

Comment:
```suggestion
If you have any feedback or suggestions about the library, start a discussion via a [GitHub](https://github.com/ballerina-platform/ballerina-standard-library/issues) issue or in the [Slack channel](https://ballerina.io/community/). Based on the outcome specification and implementation can be updated. Community feedback is always welcome. Any accepted proposal which affects the specification is stored under `/docs/proposals`. Proposals under discussion can be found with the label `type/proposal` in GitHub.
```

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-ftp/pull/538/commits/3df62c276651402b0d11a31cc5968282ed4cba5f